integration/setup/config.go

```diff
@@ -1,16 +1,174 @@
 package setup
 
 import (
+	"github.com/giantswarm/apprclient"
+	"github.com/giantswarm/aws-operator/integration/env"
+	"github.com/giantswarm/backoff"
 	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/e2e-harness/pkg/framework/filelogger"
 	"github.com/giantswarm/e2e-harness/pkg/framework/resource"
-	awsclient "github.com/giantswarm/e2eclients/aws"
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	e2eclientsaws "github.com/giantswarm/e2eclients/aws"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 )
 
+const (
+	namespace    = "giantswarm"
+	organization = "giantswarm"
+	quayAddress  = "https://quay.io"
+)
+
 type Config struct {
-	AWSClient *awsclient.Client
+	AWSClient *e2eclientsaws.Client
 	Guest     *framework.Guest
 	Host      *framework.Host
+	Release   *release.Release
 	Resource  *resource.Resource
 	Logger    micrologger.Logger
 }
+
+func NewConfig() (Config, error) {
```

Comment:
I'd like to initialize & fix stuff once. This comes in handy.

Reply:
I think this should be in the `e2e-harness` itself as kind of a framework. Everything except the `AWSClient` is going to be necessary in any e2e test. (The `Guest` stuff should be able to be empty in e2e-harness though).